dev/reference/tidb-binlog/deploy.md

```diff
@@ -569,6 +569,9 @@ The following part shows how to use Pump and Drainer based on the nodes above.
         host = "192.168.0.13"
         user = "root"
         password = ""
+        # When encrypted_password is not empty, password encrypted using
+        # `./binlogctl -cmd encrypt -text string` will be ignored
```

Comment:
Please confirm with @july2993 whether the Chinese version mean:

**`encrypted_password` is encrypted using `./binlogctl -cmd encrypt -text string`.**
**When `encrypted_password` is not empty, the `password` above will be ignored.**

Reply:
And update all other descriptions in this PR if necessary. Thanks!